Packs/CommonScripts/Scripts/FileEnrichment/README.md

```diff
@@ -0,0 +1,83 @@
+This script gathers file reputation data from multiple integrations and returns a "FileEnrichment" object with consolidated information to the context output.
+
+## Script Data
+
+---
+
+| **Name** | **Description** |
+| --- | --- |
+| Script Type | python3 |
+| Cortex XSOAR Version | 6.10.0 |
+
+## Inputs
+
+---
+
+| **Argument Name** | **Description** |
+| --- | --- |
+| file_hash | Hash of the file. Supported types are: MD5, SHA1, SHA256, and SHA512. |
+| enrichment_brands | Integrations brands to use for running the file reputation command. If not provided, the command will run for all available integrations that contain the file reputation command.<br/>For multi-select, provide a comma-separated list of integration internal names. For example: "VirusTotal \(API v3\),WildFire-v2". |
+| verbose | Whether to retrieve a human-readable entry for every command. When set to false, human-readable will be containing only summary of the final result. Default is false. |
```

Comment:
Whether to retrieve a human-readable entry for every command. When set to false, human-readable will only summarize the final result.